README.md

```diff
@@ -1931,7 +1865,7 @@ Name | Code | Message
 `ConnectionError` | ENOTOPEN | Connection not yet open.
 `ConnectionError` | EINSTLOOKUP | Instance lookup failed.
 `ConnectionError` | ESOCKET | Socket error.
-`ConnectionError` | ECONNCLOSED | Connection is closed.
+`ConnectionError` | ECONNCLOSED | Connection is closed. See [global pool single instance](#global-pool-single-instance).
```

Comment:
I'm not sure how this link is helpful. The connection can be closed for any multitude of reasons and will pretty much never be to do with single pool instances.